### ISSUE TITLE
Middle Cerebral Artery Stroke

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6856]
**Preferred term label:**
Middle Cerebral Artery Stroke

**Synonyms**
MCA

**Definition (free text, please give PubMed ID)**
Sudden impairment of blood flow to a part of the brain due to occlusion or rupture of the middle cerebral artery to the brain.
PMID: 32310592

**Parent term (use hpo.jax.org/app)**
Stroke

**Comment by pnrobinson** [comment]
adding term with definition Acute onset neurological deficits such as weakness, dizziness, numbness, issues with speech, or visual changes related to sudden impairment of blood flow to a part of the brain due to occlusion or rupture of the middle cerebral artery to the brain.
